libs/converter-nh-accuvote/src/convert/convert_election_definition.ts

```diff
@@ -3,257 +3,208 @@ import {
 
-    if (frontExpectedPaperSize !== paperSize) {
-      success = frontExpectedPaperSize === backExpectedPaperSize;
+    if (
+      frontGridAndBubbles.grid.geometry.ballotPaperSize !==
+        backGridAndBubbles.grid.geometry.ballotPaperSize ||
+      frontGridAndBubbles.grid.geometry.ballotPaperSize !==
+        convertHeader.result.election.ballotLayout.paperSize
+    ) {
       issues.push({
         kind: ConvertIssueKind.InvalidTemplateSize,
-        message: `Template images do not match expected sizes. The XML definition says the template images should be "${paperSize}", but the template images are front="${frontExpectedPaperSize}" and back="${backExpectedPaperSize}".`,
-        paperSize,
-        frontTemplateSize: {
-          width: frontPage.width,
-          height: frontPage.height,
-        },
-        backTemplateSize: {
-          width: backPage.width,
-          height: backPage.height,
-        },
-      });
-      paperSize = frontExpectedPaperSize;
-    }
-
-    if (frontGridAndBubbles.metadata && backGridAndBubbles.metadata) {
-      if (frontGridAndBubbles.metadata.side !== 'front') {
-        success = false;
-        issues.push({
-          kind: ConvertIssueKind.InvalidTimingMarkMetadata,
-          message: `front page timing mark metadata is invalid: side=${frontGridAndBubbles.metadata.side}`,
-          side: 'front',
-          timingMarkBits: frontGridAndBubbles.metadata.bits,
-          timingMarks: frontGridAndBubbles.grid.partialTimingMarks,
-        });
-      }
-
-      if (backGridAndBubbles.metadata.side !== 'back') {
-        success = false;
-        issues.push({
-          kind: ConvertIssueKind.InvalidTimingMarkMetadata,
-          message: `back page timing mark metadata is invalid: side=${backGridAndBubbles.metadata.side}`,
-          side: 'back',
-          timingMarkBits: backGridAndBubbles.metadata.bits,
-          timingMarks: backGridAndBubbles.grid.partialTimingMarks,
-        });
-      }
-    }
-
-    if (!success) {
-      return err({
-        issues,
-        election,
+        message: 'Template images do not match expected sizes.',
+        paperSize: convertHeader.result.election.ballotLayout.paperSize,
+        frontTemplateSize: frontGridAndBubbles.grid.geometry.canvasSize,
+        backTemplateSize: backGridAndBubbles.grid.geometry.canvasSize,
       });
     }
 
     const ballotStyle = election.ballotStyles[0];
     assert(ballotStyle, 'ballot style missing');
 
-    const gridLayout = election.gridLayouts?.[0];
-    assert(gridLayout, 'grid layout missing');
-
-    const frontMetadata = frontGridAndBubbles.metadata;
     const ballotStyleParty = getPartyForBallotStyle({
       ballotStyleId: ballotStyle.id,
       election,
     });
-    const partyPrefix = ballotStyleParty ? `${ballotStyleParty.abbrev}-` : '';
-    const cardNumber =
-      frontMetadata?.side === 'front' ? frontMetadata.cardNumber : 1;
-    const ballotStyleId = `${partyPrefix}card-number-${cardNumber}`;
```

Comment:
Removing this broke interpretation of timing mark ballots. The problem is that this logic is essentially duplicated in the ballot interpreter code. Both assume that for a given election only one ballot style will use a given card number, and that the ballot style ID can be generated knowing _only_ that card number. With the changes in this branch, that's no longer true. We'll need to decide what we want to do about this.

Reply:
Could we use an incrementing index for card number when generating the ballot style ids?

Reply:
We could, but it wouldn't help since they wouldn't necessarily correspond to the `cardNumber` from the timing mark-encoded metadata.